onnxscript/ir/_convenience.py

```diff
@@ -368,3 +368,29 @@ def tensor(
             doc_string=name,
         )
     return tensor_
+
+
+def create_value_mapping(graph: _core.Graph) -> dict[str, _core.Value]:
+    """Return a dictionary mapping names to values in the graph.
+
+    The mapping does not include values from subgraphs.
```

Comment:
Maybe it's useful to consider how to handle subgraphs? Or, to be more precise, outer graphs? Eg.: if we are given a subgraph, should we construct a dictionary that includes outer scope names visible?

But it's not a priority, since we don't have a specific need yet.

Reply:
Good point thanks! Currently we don't have a pointer from subgraphs to outer graphs yet. It's something I am thinking to add.